data_models/onepassword_signinattempt_data_model.py

```diff
@@ -0,0 +1,13 @@
+import panther_event_type_helpers as event_type
+
+
+def get_event_type(event):
+    failed_login_events = ["credentials_failed", "mfa_failed", "modern_version_failed"]
```

Comment:
modern version? o.O

Reply:
Yeah I kinda like that. "Your password is good, but you're client is too old so no sign-in for you"